src/importTest/index.ts

```diff
@@ -0,0 +1,16 @@
+import assert from "assert";
+
+import factories from "@dsnp/activity-content";
+import errors from "@dsnp/activity-content";
+import validation from "@dsnp/activity-content";
```

Comment:
So these all work, but they are not testing what you think they are testing. They are just testing the same thing with different names. Here are some different 
```suggestion
import activityContent from "@dsnp/activity-content";
import { factories }  from "@dsnp/activity-content";
import * as factoriesDirect from "@dsnp/activity-content/factories";
import * as errors from "@dsnp/activity-content/errors";
import * as validation from "@dsnp/activity-content/validation";
```